README.md

```diff
@@ -31,30 +37,21 @@ angular
   .module('myApp', [
     ngOvhChatbot,
   ]);
+```
 
-// By default, the chatbot hit the URL at "/chatbot".
-// If you want to use a custom url:
+## Config fields
 
-angular
-  .module('myApp')
-  .config(/* @ngInject */(ChatbotService) => {
-    ChatbotService.setChatbotUrl('/url/to/the/chatbot');
-  });
-```
+| Key        | Value            | Default       | Description                                   |
+| ---------- | ---------------- | ------------- | --------------------------------------------- |
+| space      | Any string       | "Default/Web" | Provide the knowledge space for the bot       |
+| universe   | Any string       | "WEB"         | Provide the universe for the bot              |
+| language   | Two letters code | "fr"          | Provide the language for the bot              |
+| subsidiary | Two letters code | "FR"          | Provide the country for the bot               |
 
 ## Tips
 
 * The chatbot div is automatically placed in the bottom right of the page.
 
-### ovh-angular-user-pref:
-
-Key used: `CHATBOT_PREF`
```

Comment:
Why removing this part?

Because by removing this section we lost the information about which key is used from `/me/preferences` Isn't it?

Reply:
Well actually we don't use the preferences anymore, so this was left behind.